ScorecardsOCR/Fun_ScorecardsOcr.php

```
@@ -0,0 +1,129 @@
+ *  - pl_ocr_install()         : Auto-create PLOcrConfig table if absent
+ *  - pl_ocr_get_config()      : Read a config value
+ *  - pl_ocr_save_config()     : Write a config value
+ *  - pl_ocr_lookup_scores()   : Fetch QuD{N} scores from Qualifications by bib + session
+ */
+
+/**
+ * Auto-install PLOcrConfig table if it does not yet exist.
+ */
+function pl_ocr_install(): void
+{
+    $rs = safe_r_sql("SHOW TABLES LIKE 'PLOcrConfig'");
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $rs. Configured minimum length is 3. (undefined)

(ShortVariable)